workload-team/cluster-stamp.bicep

```diff
@@ -161,6 +161,71 @@ resource targetVirtualNetwork 'Microsoft.Network/virtualNetworks@2023-11-01' exi
 
 /*** RESOURCES ***/
 
+// An Azure Monitor workspace where cluster metrics related to Prometheus are collected
+resource amw 'Microsoft.Monitor/accounts@2023-04-03' = {
+  name: 'amw-${clusterName}'
+  location: location
+}
+
+resource dce 'Microsoft.Insights/dataCollectionEndpoints@2023-03-11' = {
+  name: 'MSProm-${location}-${clusterName}'
+  location: location
+  kind: 'Linux'
+  properties: {
+    networkAcls: {
+      publicNetworkAccess: 'Enabled'
+    }
+  }
+}
+
+resource dcr 'Microsoft.Insights/dataCollectionRules@2023-03-11' = {
+  name: 'MSProm-${location}-${clusterName}'
+  kind: 'Linux'
+  location: location
```

Comment:
DCRs support logging the "Log Errors" channel to log analytics, please connect this DCR's `allLogs` group to our LAW

Reply:
great suggestion @ckittel 

done | addressed from 1b79f05f7db79fd093b676ce344a2f17df519734